puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-mysql-systest.yaml

```diff
@@ -0,0 +1,33 @@
+- job:
+    name: foreman-mysql-systest
```

Comment:
Is this nightly as well or ran against releases as well?

Reply:
I think @ehelms is asking if `repo_environment` should be `staging`

Reply:
Also asking if the name should include `nightly`